setup.py

```diff
@@ -16,8 +16,9 @@
 if sys.version_info < (3, 4):
     install_req += ["enum"]
 
+exec(open('odml/version.py').read())
```

Comment:
Using `exec` is generally a bad idea. There's no reason for that here, especially since you can just `from odml.version import __version__`.